siriuspy/siriuspy/devices/ids.py

```diff
@@ -300,6 +291,20 @@ def cmd_move_disable(self, timeout=None):
         """Command to disable and break ID phase and gap movements."""
         return self.cmd_move_phase_disable(timeout=timeout)
 
+    # --- cmd_wait
+
+    def cmd_wait_while_busy(self, timeout=None):
+        """Command wait within timeout while ID control is busy."""
+        return True
```

Comment:
is this method "to be implemented"?

Reply:
no. this method is immaterial for PAPU but makes sense for derived class EPU. I included this method in PAPU so that I did not hake to overwrite other methods for EPU.